benchopt/runner.py

```diff
@@ -338,7 +345,10 @@ def run_benchmark(benchmark, solver_names=None, forced_solvers=None,
     else:
         save_file = output_dir / f"{output_name}.csv"
         save_file = uniquify_results(save_file)
-    df.to_csv(save_file)
+    with open(save_file, 'a') as f:
```

Comment:
I used `'a'` because I first write the tag, then the content of `df`.

In principle the file should not exist (using `output` we add numbers to save_file) unless two benchmarks finish at the exact same time. If that is the current with the current design we overwrite the first one

I can add a check that the file does not exist, but it's unclear to me what to do if it exists

Reply:
`a` only change something at opening, not if you have multiple writes. (see https://docs.python.org/3/library/functions.html#open)

Basically, `w` opens the file with cursor at the beginning truncating the file content if it exists while `a` put the cursor at the end of the content.
But as long as you keep it open, the cursor is at the end of the previous write.

Reply:
of course yes!